.infra/Pulumi.prod.yaml

```diff
@@ -65,6 +65,8 @@ config:
       secure: AAABAI/bDI+xz5PMlMZseI2kW5oanHk3z1KcAOZJb+/G8arhgSP1EMdqdOTk0kku1t6KTWtl9+oSSDKWYNH1CG5D+Ac1ZPFAWc/ldmEb/iU=
     onesignalAppId:
       secure: AAABALTizkxZkCfsq6/fx2v7nXxsVP7psWBfSUct0zeyrX0ABxPpD30zIsyx5r5wqt1UJIIXDIVOnhM7W3vzrzaatrE=
+    otelTracesSampler: parentbased_traceidratio
+    otelTracesSamplerArg: 1
```

Comment:
100% is too much I think. Let's start with 10% and see what happens

Reply:
100% is probably too much, but I was thinking to keep it at 100% for a little while (an hour or so), to see that it works, and then reduce it

Reply:
Ok, will require you more attention then